.browserslistrc

```diff
@@ -1,6 +1,44 @@
-ie 11
-edge >= 14
-firefox >= 52
-chrome >= 49
-safari >= 10
-node 8.0
+[modern]
+last 1 chrome version
+last 1 edge version
+last 1 firefox version
+last 1 safari version
+node 14
+
+# snapshot of "> 0.5%, last 2 versions, Firefox ESR, not dead, not IE 11" when the last major is released.
```

Comment:
Widening to `0.2%` (which is what CRA is using; 0.5 is browserslist default) would include:
- chrome 49
- edge 18
- ios_safari 9.3

Chrome and edge are evergreen so I think we can ignore these versions. Older Chrome versions used to be relevant since it was what google used to crawl sites but [they since upgraded and are doing so more often](https://webmasters.googleblog.com/2019/10/updating-user-agent-of-googlebot.html). 

ios_safari is not because it's tied to the OS and therefore the device. The devices that support iOS 9 but not iOS 12 are:

- iPod Touch (5th generation)
- iPhone 4S
- iPhone 5
- iPhone 5C
- iPad Mini (1st generation)
- iPad 2
- iPad (3rd generation)
- iPad (4th generation)
- iPad Air
- iPad Pro (12.9-inch 1st generation)

Using https://en.wikipedia.org/wiki/IOS_9#Supported_devices and https://en.wikipedia.org/wiki/IOS_12#Supported_devices

<!-- using ios_safari 9.3 legacy would only add

transform-template-literals
transform-typeof-symbol
transform-unicode-escapes
transform-shorthand-properties
transform-literals
transform-duplicate-keys
transform-computed-properties
-->